### ISSUE TITLE
Big issue with latest Firefox 62.0

### ISSUE THREAD

**Issue by ghost** [Lusito/forget-me-not#98]
Forget Me Not ver. 1.0.5 / Firefox 62.0 (x64) / Win7 (x64)

After a clean install of Firefox 62.0 I was quite surprised to notice that FMN's toolbar button was greyed-out. I had a look in about:addons and FMN was not shown ads disabled. I had a look at the Browser console where I found this info:

`addons.webextension.forget-me-not@lusito.info WARN Firefox does not support dataTypes: serverBoundCertificates.`

Anyone have an idea of the problem?
No need to say I reverted to Firefox 61.0.2: I need Forget-Me-Not

**Comment by StanGets** [comment]
I got FF62 from https://ftp.mozilla.org/pub/firefox/releases/

No, I wasn't in Private Mode, for sure. I include in user.js,
`user_pref("browser.privatebrowsing.autostart", false);`

It was on FF launch, icon is quickly grayed out, browser console states what I mentioned above, and FMN is really not working, I checked easily by visiting a site delivering cookies.

Moreover, but this is not related to FMN, my Stylus FF extension lost all of its stylings, which seems caused by an indexedDB problem.

At this time FF62 is not usable here. That's now your problem anyway. My point is FMN being hooked by Firefox.
